.codecov.yml

```diff
@@ -0,0 +1,13 @@
+# prevent codecov from blocking a PR and remove comments
+coverage:
+  status:
+    project:
+      default:
+        informational: true
+    patch:
+      default:
+        informational: true
+
+comment: false
+github_checks:
+  annotations: false
```

Comment:
Thanks for removing the annotations on the PRs. That make it difficult to follow a review!!❤️